### DISCUSSION TITLE
what is a code of conduct

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#28] [Q&A] [answered]
As the question describes

**Comment by andatoshiki** [comment, marked as answer]
# Code of Conduct

A code of conduct is the most common policy within an organization. This policy lays out the company’s principles, standards, and the moral and ethical expectations that employees and third parties are held to as they interact with the organization. A code of conduct is an integral part of compliance efforts as it provides documentation that an employee or third party has violated company policy if illegal activity arises. Let’s explore what a code of conduct means and the impact it can have on an organization. 

## What is a Code of Conduct in the Workplace?

A company’s code of conduct is a policy that outlines principles and standards that all employees and third parties acting on behalf of the company must follow. The code of conduct reviews the organization’s mission and values and ties these ideals to professional behavior standards. In many workplaces, codes of conduct become benchmarks of performance. 

## Why is Having a Code of Conduct Important?

A code of conduct serves as a reference point for employees to make better choices on a day-to-day basis. While every possible ethical dilemma an employee might encounter won’t be spelled out, the code should lay out the guiding principles by which employees should act and therefore lead there workforce to make the right decision. 

Having a strong, ethical code of conduct is essential to building a culture of compliance throughout an organization. A code of conduct is an excellent exercise to focus the leadership team on how employees should behave at work and the standards they should uphold. These standards can have massive impacts on how the organization functions, how employees conduct themselves daily, and how the workforce interacts with others on behalf of the organization. 

On top of ethical reasons, there are legal reasons for implementing a code of conduct as well. All public organizations in the U.S. are required by law to have a code of conduct in place. Private organizations would be smart to take note of this as well. 

## What Should a Code of Conduct Include

There are a few common elements that every code of conduct should feature. An ethical code of conduct should include a letter from the CEO, reiterate the company’s values, and outline how violations are handled.

The letter from the CEO should emphasize the organization’s commitment to these standards. The note is an opportunity to express the leadership team’s prioritization of compliance and ethics. 

The code of conduct is a great place to drive the organization’s values home with employees and third parties because they will be signing and therefore agreeing to uphold these standards. Selecting your organization’s values is a critical step in building a flourishing business, and establishing a culture of compliance. 

Finally, a code of conduct should inform how violations of the code of conduct are handled internally at the organization and mention the external legal risks. The code should also review the proper channels for reporting misconduct if out-of-line behavior is witnessed. 

## What is an Effective Code of Conduct in an Organization?

Best-in-class codes of conduct have a few traits in common. 

First, they are regularly reviewed. Updating the organization’s code of conduct on an annual basis ensures that the content is up-to-date and relevant as things are always changing within organizations. The code should be a living, breathing document that is highly relevant to employees and their work. 

Second, the signatures are properly managed by a policy deployment solution that tracks signatures and time stamps dates. You can write the best code of conduct in the world, but if you don’t correctly deploy it to your employees and third parties, what was the point? Strive for your employees and third parties to sign the code of conduct and understand the contents and implications. 

Lastly, an effective code of conduct is digestible by the audience it is intended for. It is not inundated with legal speak that only the lawyers at the company can understand but instead written in an uncomplicated format that is easy for all to comprehend. While this may seem like a simple point to emphasize, its impact on the adoption and impact of the code of conduct within an organization can not be overstated.
